code/modules/cargo/packs.dm

```diff
@@ -116,6 +116,17 @@
 					/obj/item/clothing/gloves/color/latex/nitrile)
 	crate_name = "bio suit crate"
 
+/datum/supply_pack/emergency/cola_bot
```

Comment:
Also might be worth making these only come with one bot, since cargo is profiting off of it.